### ISSUE TITLE
ARM64 support

### ISSUE THREAD

**Issue by romikoops** [russellcardullo/terraform-provider-pingdom#110]
Hey, more and more developers are moving to Apple M1 (ARM64). Unfortunately, the provider does not support it.

```
│ Error: Incompatible provider version
│ 
│ Provider registry.terraform.io/russellcardullo/pingdom v1.1.3 does not have
│ a package available for your current platform, darwin_arm64.
│ 
│ Provider releases are separate from Terraform CLI releases, so not all
│ providers are available for all platforms. Other versions of this provider
│ may have different platforms supported.
```

**Comment by pointfreesubscriptions** [comment]
i have the same issue

**Comment by vodila-privet** [comment]
i have the same issue

**Comment by jon-whit** [comment]
@russellcardullo I noticed that `GOOS=darwin` and `GOARCH=arm64` is included in the goreleaser manifest (see [.goreleaser.yml L22-L27](https://github.com/russellcardullo/terraform-provider-pingdom/blob/master/.goreleaser.yml#L22-L27)), but I don't see darwin arm64 build artifacts in the releases.

Is there any way you could quickly resolve that and get a released published for darwin arm64? My team uses this terraform module but half of the team cannot use it on their machines because of the arch differences.

**Comment by aluca-private** [comment]
yes please! same issue here, all people with M1 Macs hit this wall on most workspaces

**Comment by devenkulkarni** [comment]
Hitting same issue

**Comment by romikoops** [comment]
Please prioritize the issue. Currently, it does not allow to execute terraform scripts on Mac M+
